content/intro/pricing/whats-included.md

```diff
@@ -0,0 +1,35 @@
+---
+menu:
+  main:
+    parent: pricing
+title: What's included
+weight: 10
+---
+
+All cloud.gov access packages include the following: 
+
+- API access, CLI access, Web UI access
+- Documentation/tutorial material (including continuing development)
+- Web-based support during business hours (platform available 24/7, problems addressed as-possible).
+- Allocation of underlying IaaS (AWS) instances 
+- OS security updates and regular hardening of OS image
+- Regular updates to platform security
+- Network security
+- ATO-ready documentation of system components (compliance-masonry form)
```

Comment:
Maybe "via [Compliance Masonry](https://github.com/opencontrol/compliance-masonry)" (with link). Also, would "ATO-ready" be overselling it a bit? Should it say something to the effect of "Documentation to ease your agency's Authority to Operate process"?

/cc @geramirez